webpack/renderer.ts

```diff
@@ -103,14 +109,18 @@ export function webpackLensRenderer({ showVars = true } = {}): webpack.Configura
         cwd: __dirname,
         exclude: /node_modules/,
         failOnError: true,
-      }),
+      }) as unknown as WebpackPluginInstance,
 
       new MiniCssExtractPlugin({
         filename: "[name].css",
       }),
 
-      isDevelopment && new ReactRefreshWebpackPlugin(),
-    ].filter(Boolean),
+      ...(
+        isDevelopment
```

Comment:
Is this ternary the wrong way around? `ReactRefreshWebpackPlugin` should be present if `isDevelopment`?

Reply:
Good catch